### PR TITLE
Add hiccup notebook

### LINKED ISSUE #9: Hiccup
As a researcher I am interested in detecting hiccup I want to generate some time series.
The scenario I want to model is
* Hiccups should be observed for a week, except during sleeping (23:00 - 8:00).
* A hiccup happens once an hour at a random time
* Time series should have a time point every minute

Other scenarios that would nice
1. Multiple hiccups occur in a bout (see https://en.wikipedia.org/wiki/Hiccup for histogram)
1. Person has infrequent multiple hiccups once a day
1. Person has higher chance of hiccup after eating (from 8:00 - 9:00, 12:00 - 13:00 and 18:00 - 19:00)

---
With this notebook I want to show usage of a time series which is non-continuous and random locations.

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #9

### CODE REVIEW

File: hiccup.ipynb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    "\n",
+    "hiccups = np.zeros(len(timepoints), dtype=float)\n",
+    "for day in range(5):\n",
+    "    daily_offset = day * (24 * 60)\n",
+    "    for hour in range(15):\n",
+    "        parameters = space.sample()\n",
+    "        # add offset, watchout placement should pick a value from timepoints, not an index\n",
+    "        parameters['placement'] = floor(parameters['placement']) + daily_offset + hour * 60\n",
+    "        hiccup = triangular_peak(timepoints, width_base_left=0, width_base_right=2, **parameters) # TODO unable to make peak of 1 time point\n",
+    "        hiccups += hiccup\n",
```
Comment (eriktks) [inline hiccup.ipynb:73]:
```suggestion
    "        hiccups[parameters['placement']] = parameters['height']\n",
```
This change will generate a peak of width zero

**Review by eriktks — APPROVED** [review]
I could not run the notebook with Python 3.6 but it worked with Python 3.7. The generated graph was strange because there was a hiccup at 08:00 which resulted in a diagonal line from the previous date 22:59 (value 0) to that day 08:00 (value > 0). This could be avoided by not allowing hiccups at 08:00 and 22:59 or setting the values for 07:59 and 23:00 explicitly at 0. All else was fine.

File: hiccup.ipynb (hunk 1 above)
Comment (sverhoeven) [reply hiccup.ipynb:73]: I want the package to fix it, not do a workaround here.

**Comment by sverhoeven** [comment]
Thanks for reviewing. Having a line plot is probably not the best, running again can give you a nicer plot.

**Pull request merged**